front_end/src/server.js

```diff
@@ -0,0 +1,33 @@
+var express = require('express');
+var path = require('path');
+var httpProxy = require('http-proxy');
+
+var proxy = httpProxy.createProxyServer().listen(3000);
```

Comment:
This is weird. I didn't have to create a proxy: https://github.com/rlr/webtiles-previewer-redux/blob/master/server.js

That being said, we might need a proxy to send the API requests to python/flask.